### DISCUSSION TITLE
Using sccache with bazel rust

### DISCUSSION THREAD

**Discussion by kesavkolla** [bazelbuild/rules_rust#1918] [Q&A]
Is it possible to use sccache for caching the bazel build which is using rust project?

**Comment by illicitonion** [comment]
You can't use sccache, but Bazel has support for its own [remote caching](https://bazel.build/remote/caching) and [remote execution](https://bazel.build/remote/rbe) systems.

At my company, I'm using Bazel's remote execution support with rules_rust with good success.
